build/wix/mixxx.wxs

```diff
@@ -0,0 +1,429 @@
+  -->
+ 
+  <!--
+    ====================================================================================
+    Defines & Variables
+  -->
+  
+  <!-- Full version number to display -->
+  <?define VersionNumber="!(bind.FileVersion.mixxxExe)" ?>
+ 
+  <!--
+  Upgrade code HAS to be the same for all updates.
+  Once you've chosen it don't change it.
+  !!! DO NOT REUSE IN ANY OTHER PROJECT !!!
+  -->
+  <?define UpgradeCode="921DC99C-4DCF-478D-B950-50685CB9E6BE" ?>
+ 
+  <!-- The URL for add/remove programs -->
+  <?define InfoURL="http://mixxx.org/" ?>
+   
+  <!-- 32-bit / 64-bit variables -->
+  <?if $(var.Platform) = x64 ?>
+    <?define Win64 = "yes" ?>
+    <?define arch = "64" ?>
+    <?define notarch = "32" ?>
+    <?define PlatformProgramFilesFolder = "ProgramFiles64Folder" ?>
+    <?define baseBuildDir = "..\.." ?>
+  <?else ?>
+    <?define Win64 = "no" ?>
+    <?define arch = "32" ?>
+    <?define notarch = "64" ?>
+    <?define PlatformProgramFilesFolder = "ProgramFilesFolder" ?>
+    <?define baseBuildDir = "..\.." ?>
+  <?endif ?>
+ 
+  <?define SourceDir="$(var.baseBuildDir)\dist$(var.arch)"?>
+  
+  <!--
+    ====================================================================================
+    Package start
+  -->
+
+  <!-- The upgrade code must never change as long as the product lives! -->
+  <!-- Product IDs must be autogenerated (*) or else major upgrades will not work -->
+  <Product Id="*"
+           Name="!(loc.ApplicationName)"
+           Language="!(loc.Language)"
+           Version="$(var.VersionNumber)"
+           Manufacturer="!(loc.ManufacturerFullName)"
+           UpgradeCode="$(var.UpgradeCode)" >
+   
+    <!-- Package IDs are valid for a single package version only - they are autogenerated by WiX -->
+    <!-- Let's require Windows Installer 4.0 (included in Vista) -->
+    <!-- And ALWAYS install per machine!!! -->
+    <Package Id="*"
+             Keywords="Installer"
+             InstallerVersion="400"
+             Compressed="yes"
+             InstallScope="perMachine"
+             Description="!(loc.ProductDescription)"
+             Manufacturer="!(loc.ManufacturerFullName)"
+             Comments="!(loc.Comments) $(var.VersionNumber)"
+             Platform="$(var.Platform)" />
+
+    <!-- Define icons (ID should not be longer than 18 chars and must end with ".exe") -->
+    <Icon Id="Icon.exe" SourceFile="$(var.baseBuildDir)\res\images\ic_mixxx.ico" />             
+    <Icon Id="Manual.ico" SourceFile="images\manual.ico" />             
+    
+    <!-- Set properties for add/remove programs -->
+    <Property Id="ARPPRODUCTICON" Value="Icon.exe" />
+    <Property Id="ARPHELPLINK" Value="$(var.InfoURL)" />
+    <Property Id="ARPNOREPAIR" Value="yes" Secure="yes" />      <!-- Remove repair -->
+    <Property Id="ARPNOMODIFY" Value="yes" Secure="yes" />      <!-- Remove modify -->
```

Comment:
I can see use-case too, but I'm not currently confortable enough with that to enable them. So this will be in my todo list ;)